src/__tests__/sdk_codes.test.ts

```diff
@@ -0,0 +1,53 @@
+import { Logger } from '../log';
+import * as sdkCodes from '../sdk_codes';
+
+describe('SDK Tests', () => {
+  let logger: Logger;
+
+  beforeEach(() => {
+    logger = {
+      trace: jest.fn(),
+      debug: jest.fn(),
+      info: jest.fn(),
+      warn: jest.fn(),
+      error: jest.fn(),
+    };
+  });
```

Comment:
No need to put this in a `beforeEach` since you don't assert on it and it's never modified.
```suggestion
  const logger: Logger = {
    trace: jest.fn(),
    debug: jest.fn(),
    info: jest.fn(),
    warn: jest.fn(),
    error: jest.fn(),
  };
```

Reply:
Wow nice... committing this change